scripts/amd/gemm/matmul.py

```diff
@@ -37,15 +39,21 @@ def prune_configs(configs, named_args):
             continue
         if SIZE_N <=32 and BLOCK_SIZE_N != 32:
             continue
+        if BLOCK_SIZE_M * BLOCK_SIZE_K / config.num_warps / 64 <= 2:
```

Comment:
These are for mixed input or regular gemm?

Reply:
They are for fp8.